src/Bicep.Core/Semantics/ParamSemanticModel.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Bicep.Core.Diagnostics;
+using Bicep.Core.Syntax;
+using Bicep.Core.Workspaces;
+
+namespace Bicep.Core.Semantics
+{
+    public class ParamSemanticModel
```

Comment:
The new parser is called `ParamsParser`. Should we change this to `ParamsSemanticModel` for sake of consistency?

Reply:
Sure!